docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -129,22 +129,16 @@ type Props = OwnProps &
   }>;
 
 const MethodPage: React.FunctionComponent<Props> = (props) => {
-  const service = props.specification.getServiceByName(
-    props.match.params.serviceName,
-  );
+  const params = props.match.params;
+  const service = props.specification.getServiceByName(params.serviceName);
   if (!service) {
     return <>Not found.</>;
   }
-
-  const method = service.methods.find(
-    (m) =>
-      m.name === props.match.params.methodName &&
-      m.httpMethod === props.match.params.httpMethod,
-  );
+  const id = `${params.serviceName}/${params.methodName}/${params.httpMethod}`;
```

Comment:
Should we include `overloadId` here as well then?

Reply:
No, we shouldn't. 😉 
`params.methodName` is the path parameter that is routed using the Router.
https://github.com/line/armeria/blob/6bd02dc74ba924ab51740f81adffefccd0351d8e/docs-client/src/containers/App/index.tsx#L391
It could be `overload-1` if a user types or navigates through the link.